libs/components/pages/src/lib/modules/modal-link-list/modal-link-list.component.html

```diff
@@ -0,0 +1,41 @@
+<ng-container
+  *ngIf="links === 'loading'; then loadingTemplateRef; else linksTemplateRef"
+></ng-container>
+
+<ng-template #headingTemplateRef>
+  <h2 class="sky-font-heading-4 sky-margin-stacked-sm">
+    {{ title }}
+  </h2>
+</ng-template>
+<ng-template #loadingTemplateRef>
+  <sky-wait [isWaiting]="true"></sky-wait>
+  <ng-container [ngTemplateOutlet]="headingTemplateRef"></ng-container>
+</ng-template>
+<ng-template #linksTemplateRef>
+  <ng-container *ngIf="links?.length > 0">
+    <ng-container [ngTemplateOutlet]="headingTemplateRef"></ng-container>
+    <ul
+      class="sky-modal-link-list"
+      [skyThemeClass]="{
+        'sky-margin-stacked-separate': 'default',
```

Comment:
Per @Blackbaud-AdamFunderburk - looks like the list "blocks" of links are using one the deprecated spacing classes `sky-margin-stacked-separate`

Reply:
Updated.